lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -1318,18 +1318,20 @@ namespace Js
                 IntlEngineInterfaceExtensionObject* intlExtensionObject = static_cast<IntlEngineInterfaceExtensionObject*>(nativeEngineInterfaceObj->GetEngineExtension(EngineInterfaceExtensionKind_Intl));
                 if (args.Info.Count == 2)
                 {
-                    auto undefined = scriptContext->GetLibrary()->GetUndefined();
-                    CallInfo toPass(callInfo.Flags, 7);
-                    return intlExtensionObject->EntryIntl_CompareString(function, toPass, undefined, pThis, pThat, undefined, undefined, undefined, undefined);
```

Comment:
>undefined, undefined, undefined, undefined [](start = 115, length = 42)

where did these four undefined values go?

Reply:
I handle whether we get these arguments or not in the backing function now.